docs/architecture/microservices/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -11,17 +11,17 @@ The following decision table summarizes whether to use .NET Framework or .NET Co
 > Your development machines will run one Docker host, either Linux or Windows. Related microservices that you want to run and test together in one solution will all need to run on the same container platform.
 
 | Architecture / App Type | Linux containers | Windows Containers |
-|-------------------------|------------------|--------------------|
+|-------------------------|:----------------:|:------------------:|
```

Comment:
any reason why this is not applied to all columns?

Reply:
Well, that's the format on the original word version of the e-book.

I've been centering that by hand while generating the e-book, and this is the first time the generation went so smoothly that I've had the time to fix those minor issues 😅